zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLFuncVisitor.scala

```diff
@@ -205,26 +205,37 @@ object AQLFuncVisitor {
     */
   def translateAggrFunc(field: Field,
                         func: AggregateFunc,
-                        aqlExpr: String
+                        aqlExpr: String,
+                        isAggrOnly: Boolean
                        ): (DataType.DataType, String, String, String) = {
-    val newvar = s"${aqlExpr.split('.')(0)}aggr";
+    val parameter = StringBuilder.newBuilder
+    val wrap = StringBuilder.newBuilder
+    if (isAggrOnly) {
+      parameter.append("")
+      wrap.append("")
+    } else {
+      val newvar = s"${aqlExpr.split('.')(0)}aggr";
+      parameter.append(s"$newvar")
+      wrap.append(")")
```

Comment:
since the `wrap` always contains `)`, can you remove the `)` and write the return string from
`s"count($parameter$wrap"`  to `s"count($parameter$wrap)" ? otherwise, i will be crazy to find the paired `)`